pyproject.toml

```diff
@@ -26,48 +26,48 @@ celery = {extras = ["redis"], version = "^5.1.2"}
 python = "^3.8.0"
 fastapi = "^0.68.0"
 # Per the `uvicorn docs <https://www.uvicorn.org/#quickstart>`_, install the standard (as opposed to minimal) uvicorn dependencies.
-uvicorn = {extras = ["standard"], version = "^0.14.0"}
+uvicorn = {extras = ["standard"], version = "^0.17.0"}
 # See the `poetry docs <https://python-poetry.org/docs/dependency-specification/#using-environment-markers>`_.
-gunicorn = {version = "^20.1.0", markers = "sys.platform != 'win32'"}
-Jinja2 = "^3.0.1"
```

Comment:
It is good to get this syntax cleaned up, as it was not really doing what we thought it was doing.

Reply:
Good point, thanks!